Minor fixes to the blog post Understanding ECMAScript part 1. (#328)
v8/v8.dev: src/blog/understanding-ecmascript-part-1.md

> 2. Let `O` be `? ToObject(this value)`.
> 3. Return `? HasOwnProperty(O, P)`.

> and

Reviewer: Why remove these?

```suggestion
…and…	
```

Author: I think the ...-character is a weird artifact coming from the times when I edited the blog post draft in Google Docs (it changes ... into this weird ...-character). First I thought I'll replace with ... but it's cleaner without any unnecessary dots.

Reviewer 2: AFAIK it's a character Mathias prefers and often adds instead of a regular `...` after PRs have been merged, so it might be where it's coming from :)

> 2. `ReturnIfAbrupt(obj)`
> 3. `Bar(obj)`. (If we’re still here, `obj` is the value extracted from the Completion Record.)

> And now [the question mark](https://tc39.es/ecma262/#sec-returnifabrupt-shorthands) comes into play: `? Foo()` is equivalent to `ReturnIfAbrupt(Foo())`. Using a shorthand is practical: we don’t need to write the error handling code explicitly each time.

Reviewer 2: I don't feel this explains it as well as Shu's comment you posted on Twitter. Could you add some modified version of that comment here for better clarity?

> And now [the question mark](https://tc39.es/ecma262/#sec-returnifabrupt-shorthands) comes into play: `? Foo()` is equivalent to `ReturnIfAbrupt(Foo())`. Using a shorthand is practical, so that we don't need to write the error handling code explicitly each time.

Reviewer: ```suggestion
And now [the question mark](https://tc39.es/ecma262/#sec-returnifabrupt-shorthands) comes into play: `? Foo()` is equivalent to `ReturnIfAbrupt(Foo())`. Using a shorthand is practical: we don’t need to write the error handling code explicitly each time.
```
